docs/concepts/configuration/pod-priority-preemption.md

```diff
@@ -100,7 +100,7 @@ that use the name of the deleted PriorityClass.
 ### Example PriorityClass
 
 ```yaml
-apiVersion: v1
+apiVersion: v1alpha1
```

Comment:
please double verify if this should be "`scheduling.k8s.io/v1alpha1`".

Reply:
@tengqm Yes, you are right, I will fix it.